r-getting-started.Rmd

```diff
@@ -0,0 +1,329 @@
+completely new and you don't know where to start. In that scenario, there are
+multiple internet resources that are helpful:
+
+* blogs
+* Twitter
+* tutorials
+* question pages on Stack Overflow, an open forum that allows community members
+to ask and answer questions
+
+It can be challenging to find the right words to search when you're
+troubleshooting something new. Don't be discouraged, and think about different
+ways you can apply the terminology you're learning to make your search terms
+clearer and more specific.
+
+Even when you find information that seems useful,
+these resources can be overwhelming and confusing.
+Some things to look for in a Stack Overflow discussion include:
+
+* Is the question clearly stated and is there example code? If you're
+debugging, does the example code look like yours?
+* Answers have upvotes and downvotes. Is there one clear answer that has a lot
+of upvotes in response to a question?
+* How complex is the answer? While some questions will necessarily have a
+complicated answer, for many common programming tasks, there should be a
+solution that only takes a few lines of code.
+* Do you recognize any terms in the solution? If you don't, are there other
+terms for which you could search?
+
+
+> Conventions
+>
+> One tool in using help pages (and in reading the rest of the book) will
+> be understanding the conventions that writers use when describing
+> many of the terms introduced in this chapter. For example
+>
+> `min()`
+>
+> indicates that something is a function, with parentheses to indicate
+> that it is an action with (potentially) input and output.
+>
+> Some other conventions used in this book are:
+> * `folder_name/` for a folder
+> * `variable_name` for variables
+> * `column_name` for columns
+
+## Exercises
+
+1. Run each of the functions above. Can you explain what each function
+expects as input, and what kind of output it produces?
+
+1. What would be the pros and cons of using the console versus a script in each situation?
+    - Writing a data analysis with multiple steps.
+    - Opening a new data set and exploring its dimensions.
+    - Checking the value of a variable.
+
+1. You've saved a string of interesting data to a variable.
+
+    ```{r}
+    sentence = "One fish, two fish, red fish, blue fish."
+    ```
+
+    How can you count the occurrences of the letter "f"?
+
+1. After googling "import csv file r", the two following pages appear in the
+search results:
+  https://stackoverflow.com/questions/3391880/how-to-get-a-csv-file-into-r
+  http://www.sthda.com/english/wiki/reading-data-from-txt-csv-files-r-base-functions
+    - What is useful about these two pages?
+    - Do you understand all of the code in the second page?
+    - The second page provides many possible options for code to import a
+    csv file. Where would you start in attempting to perform this task?
+
+1. Try reading a csv file:
+
+    ```{r eval=FALSE}
+    data <- read_csv("measurements.csv")
```

Comment:
Do they have access to this dataset? Where is it coming from?

Reply:
I modified this exercise from the python materials and interpreted it to be either a toy example or placeholder for a chunk of data we'd have readers use later. Any insight on this, @ChristinaLK ?

Reply:
Gotcha, take makes sense.